_data/services.yml

```diff
@@ -1,3 +1,19 @@
+sync:
+  name: Data Sync
+  icon: sync
+  link: /services/data-sync/
+  description: Add realtime data synchronization and offline capabilities to your mobile app
+  features:
+    - Use GraphQL to design the APIs for your mobile app
```

Comment:
I think that usually on those websites statements are passive. For example:

`Enables real time updates` `Leverages GraphQL`

Reply:
I was following the same format as the features for other services. If we change it here, we have to change all of them.